dtests/src/test/scala/io/snappydata/hydra/SnappyHydraRunner.scala

```diff
@@ -65,7 +65,11 @@ class SnappyHydraRunner extends SnappyTestRunner {
     val c14 = "grep -v java.lang.reflect.InvocationTargetException"
     val c15 = "grep -v org.apache.spark.storage.ShuffleBlockFetcherIterator." +
         "throwFetchFailedException"
-    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15
+    val c16 = "grep -v java.lang.IllegalStateException"
+    val c17 = "grep -v com.gemstone.gemfire.distributed.LockServiceDestroyedException"
+    val c18 = "grep -v com.gemstone.gemfire.GemFireIOException"
+    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15 #|
+        c16 #| c17 #| c18
```

Comment:
IllegalStateException can come for a whole host of reasons. Why is it expected and can be ignored in all tests? The old "ExpectedException" tag mechanism used in GemFire used to add test specific tags and region-bound to tell the region in log-file where exception can come and can be ignored by grepLogs. The same approach should be used here instead of blanket ignore. Secondly still do not understand why tests pass even though there have been a whole host of exceptions causing the Spark tasks to fail with errors.